src/Tests/Microsoft.NET.PerformanceTests/BuildPerf.cs

```diff
@@ -71,6 +71,7 @@ public void BuildWebLarge(ProjectPerfOperation operation)
             var testDir = _testAssetsManager.CreateTestDirectory("WebLarge", identifier: operation.ToString());
             Console.WriteLine($"Mirroring {sourceProject} to {testDir}...");
             FolderSnapshot.MirrorFiles(sourceProject, testDir.Path);
+            TestContext.Current.WriteGlobalJson(testDir.Path);
```

Comment:
Why is this necessary?

Reply:
So that if you run the perf tests with an `-sdkVersion` argument, it will respect that.  Most of the time the global.json will be written when the test directory is created, but in this case `MirrorFiles` wipes out anything in the target directory, so we have to explicitly write it again.